config.json

```diff
@@ -496,6 +496,11 @@
       "difficulty": 1,
       "slug": "forth",
       "topics": []
+    },
+    {
+      "difficulty": 1,
+      "slug": "change",
+      "topics": []
```

Comment:
Perhaps Recursion might apply too, and/or Searching.  One of the newer test cases fails with my example, so must re-consider my algorithm choice! Having fun with it.